### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by samuelgoto**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -212,7 +212,7 @@ Allows access to the file system.
 
 ---
 
-## [SMS Receiver API](https://web.dev/sms-receiver-api-announcement/)
+## [WebOTP API](https://web.dev/sms-receiver-api-announcement/)
```
Comment (millette) [inline README.md]: It's now at https://web.dev/web-otp/ it seems.

Comment (samuelgoto) [reply README.md]: it redirects

Comment (millette) [reply README.md]: I'm still seeing a 404 at https://web.dev/sms-receiver-api-announcement/

Comment (millette) [reply README.md]: Ah yes, after a hard reload, I see it now.

**Pull request merged**